### PR TITLE
add support to v2 REST API for additional types

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
Adding support for additional types referenced in performance testing - decimal, double, timestamp. See for example NoSQLBench http tests here: https://github.com/nosqlbench/nosqlbench/tree/main/driver-http/src/main/resources/activities/baselines

### CODE REVIEW

**Review by tatu-at-datastax — APPROVED** [review]

File: sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java [hunk 1]
```diff
@@ -204,6 +205,18 @@ public JsonNode jsonNodeFrom(QueryOuterClass.Value value) {
     }
   }
 
+  protected static final class DecimalCodec extends FromProtoValueCodec {
+    @Override
+    public Object fromProtoValue(QueryOuterClass.Value value) {
+      return Values.decimal(value);
+    }
```
Comment (olim7t) [inline sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java:212]: Do we really need this method?
It looks like every implementation is going to be a direct call to the corresponding `Values.<typeName>` method.

Comment (tatu-at-datastax) [reply sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java:212]: I did not check that, but if it really is `value.getDecimal()`, that'd be better?

**Review by mpenick — APPROVED** [review]
Looks good. Should there be some sort of `testDecimalHandling` test? Or is part of the "steel thread" to just get the current tests working?

File: sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java (hunk 1 above)
Comment (jeffreyscarpenter) [reply sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java:228]: I collapsed the method

**Comment by jeffreyscarpenter** [comment]
> Looks good. Should there be some sort of `testDecimalHandling` test? Or is part of the "steel thread" to just get the current tests working?

I extended the integration tests to leverage `decimal` types. But we probably do need to start creating unit tests for this service soon.

**Pull request merged**